gemonlinedb/Makefile

```diff
@@ -15,6 +15,7 @@ GEMONLINEDB_VER_MINOR=0
 GEMONLINEDB_VER_PATCH=0
 
 include $(BUILD_HOME)/$(Project)/config/mfDefsGEM.mk
+include $(BUILD_HOME)/$(Project)/config/mfPythonDefsGEM.mk
```

Comment:
Forgive my ignorance, why do we need Python definitions here?

Reply:
Dependencies on `gemutils`, should probably be cleaned up, but for now... c'est la vie